atomic_reactor/util.py

```diff
@@ -370,6 +371,19 @@ def get_image_upload_filename(image_type, image_id, platform):
                            platform=platform, ext=ext)
 
 
+def get_pipeline_run_start_time(conf: 'atomic_reactor.config.Configuration' = None,
```

Comment:
This method can be refactored to be better after https://github.com/containerbuildsystem/atomic-reactor/pull/1932 is merged